config/crd/bases/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -298,13 +298,18 @@ spec:
                 type: object
               namespace:
                 default: ""
-                description: Namespace  where console plugin and goflowkube pods are
+                description: Namespace where console plugin and goflowkube pods are
                   going to be deployed. If empty, the namespace of the operator is
                   going to be used
                 type: string
             type: object
           status:
```

Comment:
Should we log a message explaining the state in the status ? 
Like "Creating pods to XXX namespace"
Or "Moving from XXX namespace to YYY namespace"

Reply:
yes I can do that tomorrow

Reply:
Perfect ! Thanks